test/utils.jl

```diff
@@ -33,6 +33,7 @@
               pairwise(SqEuclidean(), X; dims=2)
         @test KernelFunctions.pairwise(SqEuclidean(), DX, DY) ≈
               pairwise(SqEuclidean(), X, Y; dims=2)
+        @test vcat(DX, DY) == ColVecs(hcat(X, Y))
```

Comment:
A small suggestion since `ColVecs(X) == collect(eachcol(X))` is `true` as well:
```suggestion
        @test vcat(DX, DY) isa ColVecs
        @test vcat(DX, DY).X == hcat(X, Y)
```